### ISSUE TITLE
Multiline descriptions not recognised

### ISSUE THREAD

**Issue by PussInDasBoot** [DmitryTsepelev/rubocop-graphql#29]
I've tried adding multiline comments in the following formats:

```ruby
class PersonType < BaseObject
  description "A Person has four limbs.
  People can ride buses and bikes."
```
and
```ruby
class PersonType < BaseObject
  description "A Person has four limbs."\
  "People can ride buses and bikes."
```
Both are not picked up by the cop as compliant. I've seen it on Objects and Arguments thus far. Can this be supported with some documentation on which way might be preferable? Thanks! CC @bessey

**Comment by DmitryTsepelev** [comment]
Hi @PussInDasBoot! Looks like a bug in the cop

**Comment by DmitryTsepelev** [comment]
Fixed in #31 and 0.6.2
